### ISSUE TITLE
Duplicate axioms/wrong data in WorldAirports.kif

### ISSUE THREAD

**Issue by jefft0** [ontologyportal/sumo#26]
WorldAirports.kif has dozens of duplicate axioms (as reported when Sigma loads). For example:

```
(instance LakeHavasuCityAZAirport Airport)
(abbreviation "HII" LakeHavasuCityAZAirport)
(located LakeHavasuCityAZAirport UnitedStates)

(instance LakeHavasuCityAZAirport Airport)
(abbreviation "LHU" LakeHavasuCityAZAirport)
(located LakeHavasuCityAZAirport UnitedStates)
```

Sigma is actually complaining about the duplicate axioms for "instance" and "located" by my concern is the redefinition of the airport abbreviation. "HII" seems correct https://en.wikipedia.org/wiki/List_of_airports_by_IATA_code:_H#HI but Wikipedia says "LHU" is for a different airport https://en.wikipedia.org/wiki/List_of_airports_by_IATA_code:_L#LH. Other cases have similar problems.

What is the data source for WorldAirports.kif. Maybe it needs to be refreshed? Maybe directly from Wikidata? https://www.wikidata.org/wiki/Special:WhatLinksHere/Property:P238

**Comment by arademaker** [comment]
I would definitely vote to remove those data from SUMO or having a script to obtain SUMO axioms from wikidata from these ABOX cases.

**Comment by apease** [comment]
fixed per issue #49
